### ISSUE TITLE
[TK] Kaelthas stops mindcontrolling during P5 transition

### ISSUE THREAD

**Issue by balleny** [chromiecraft/chromiecraft#7012]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

Generic

### Current Behaviour

as soon as the RP for p5 transition starts, Kaelthas stops mind controlling people

### Expected Blizzlike Behaviour

can still use mind control in the transition phase (tho overall unlikely from screening dozen of videos)

### Source

https://youtu.be/Cu3tFi68vK4?t=2827

### Steps to reproduce the problem

https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/scripts/Outland/TempestKeep/Eye/boss_kaelthas.cpp

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/564192f8810b32823925c9c1644a0abfa84d9d03

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by VS22Ayrton** [comment]
[Link to generally accepted wiki, which details Kael'Thas Sunstrider abilites in P4 and P5](https://wowwiki-archive.fandom.com/wiki/Kael%27thas_Sunstrider_(tactics)#Phase_4_specific_abilities) 
Note. P4 of Kael'thas Refers to state, where Kael'Thas Personally engage you after ressurection of his advisors but before 50% hp mark *transition phase, where celling breaks and u can see netherstorm skybox*

TL;DR It is not a bug,
